src/test/java/tests/OpenMainPageCreateNewPastebin.java

```diff
@@ -0,0 +1,26 @@
+package tests;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class OpenMainPageCreateNewPastebin {
+    public static void main(String[] args) throws InterruptedException  {
```

Comment:
according to the task, it should be WebDriver + Java + TestNG. So no need of `public static void main`